app/src/main/java/org/mozilla/focus/fragment/CrashReporterFragment.kt

```diff
@@ -24,8 +25,12 @@ class CrashReporterFragment : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        TelemetryWrapper.crashReporterOpened()
+
         closeTabButton.setOnClickListener {
```

Comment:
Is there another way to close this button? by hitting back maybe? Just want to make sure you're not losing any events, and only collecting explicit clicks.